tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -19,6 +19,7 @@
 #include <mutex>
 #include <set>
 #include <regex>
+#include <api.h>
```

Comment:
Files under `src` folder should not be included by user application, only from `include` folder.
If a function needs to be exposed to users it should be done via the C API.

I am still not sure we need a new API for this.